public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -57,7 +57,7 @@ export interface DashboardLoaderState {
 export function transformSaveModelToScene(rsp: DashboardDTO): DashboardScene {
   // Just to have migrations run
   const oldModel = new DashboardModel(rsp.dashboard, rsp.meta, {
-    autoMigrateOldPanels: true,
+    autoMigrateOldPanels: false,
```

Comment:
Do you want to leave it like this?

Reply:
@dprokop yes, because then it is controlled by the feature toggle (like normal dashboards)